### ISSUE TITLE
Having Python installed breaks plugin and NetBeans

### ISSUE THREAD

**Issue by DavoDC** [wakatime/netbeans-wakatime#26]
# **Problem** 
My WakaTime plugin was preventing my NetBeans from loading. It would just hang, and no GUI elements can be clicked/used.
Several times, I reset my NetBeans (cleared User and Cache directories) and tried to install the plugin, and three different sequence of events would always happen.

My Details:
- NetBeans WakaTime 4.0.11 (I also tried NetBeans WakaTime 4.0.8 but that didn't work either)
- NetBeans 12.0
- Windows 11
- Python 3.10.1 (Desktop) (Also added to PATH)
- Python 3.9 (MS Store App)

The first log is for when I install the plugin on a fresh copy of NetBeans.
The error appears to be the one below:
```
CONFIG [WakaTime]: Executing CLI: [pythonw, C:\Users\David\AppData\Roaming\WakaTime\legacy-python-cli-master\wakatime\cli.py, --key, XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX0a8c, --file, C:/Users/David/Downloads/newJavaFile.java, --plugin, NetBeans/12.0-631bd69cd6112b1cc4c892c24e3e605b1ba04241 netbeans-wakatime/Unknown, --write]
CONFIG [WakaTime]: pythonw: can't open file 'C:\\Users\\David\\AppData\\Roaming\\WakaTime\\legacy-python-cli-master\\wakatime\\cli.py': [Errno 2] No such file or directory
CONFIG [WakaTime]: Command finished with return value: 2
INFO [org.netbeans.modules.java.hints.WrongPackageSuggestion]: source cp is either null or does not contain the compiled source cp=
```

The second log is when I restart NetBeans, trying to see if I can get the plugin to work.
```
CONFIG [WakaTime]: Executing CLI: [pythonw, C:\Users\David\AppData\Roaming\WakaTime\legacy-python-cli-master\wakatime\cli.py, --key, XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX0a8c, --file, C:/Users/David/Downloads/newJavaFile.java, --plugin, NetBeans/12.0-631bd69cd6112b1cc4c892c24e3e605b1ba04241 netbeans-wakatime/4.0.11, --write]
INFO [org.netbeans.modules.java.hints.WrongPackageSuggestion]: source cp is either null or does not contain the compiled source cp=
```

The third log is my third restart of NetBeans, and at that point, it hangs.
NetBeans can only be closed with Task Manager and all subsequent openings hang with the same error as well.
The only way to reopen NetBeans is to force uninstall WakaTime by clearing the User and Cache directories.
Perhaps python.exe needs to be used instead of pythonw.exe? I'm not sure, and I'm not sure how I would do this.
```
INFO [WakaTime]: Initializing WakaTime plugin v4.0.11 (https://wakatime.com/)
CONFIG [WakaTime]: Logging level set to DEBUG
CONFIG [WakaTime]: WakaTime Core Location: C:\Users\David\AppData\Roaming\WakaTime\legacy-python-cli-master\wakatime\cli.py
CONFIG [WakaTime]: WakaTime Core Exists: true
CONFIG [WakaTime]: WakaTime Core Location: C:\Users\David\AppData\Roaming\WakaTime\legacy-python-cli-master\wakatime\cli.py
CONFIG [WakaTime]: WakaTime Core Exists: true
CONFIG [WakaTime]: Found python binary: pythonw
# NetBeans hangs at this point
```

The fourth log is my WakaTime log throughout this.
 
 Here are all the logs:
1. [1-messagesDuringInstall.log](https://github.com/wakatime/netbeans-wakatime/files/7828582/1-messagesDuringInstall.log)
2. [2-messagesAfterRestartingNB.log](https://github.com/wakatime/netbeans-wakatime/files/7828583/2-messagesAfterRestartingNB.log)
3. [3-messagesAfterRestartingNB_Again.log](https://github.com/wakatime/netbeans-wakatime/files/7828584/3-messagesAfterRestartingNB_Again.log)
4. [dot_wakatime_log.log](https://github.com/wakatime/netbeans-wakatime/files/7828586/dot_wakatime_log.log)

# **Resolution**
After doing research, I found [this](https://github.com/wakatime/jetbrains-wakatime/issues/120) and it gave me the idea to **uninstall both my Desktop and Store Python**.
After doing this, the plugin downloaded its own Python and it finally worked!

# **Remarks**
Even though I solved this issue, I wanted to post this for others, because I wasted a couple of hours on this!

By the way, installing Desktop or Store Python _after_ WakaTime was successfully installed and working breaks it again, so I cannot have Python on my PC if I want WakaTime NetBeans. 
The log shows that NB hangs at this line, like before
`CONFIG [WakaTime]: Found python binary: pythonw`
Uninstalling Python lets the plugin and NB load right away.

The plugin works fine when using its own Python (see below). Perhaps it could be configured to always use this.
```
INFO [WakaTime]: Initializing WakaTime plugin v4.0.11 (https://wakatime.com/)
CONFIG [WakaTime]: Logging level set to DEBUG
CONFIG [WakaTime]: WakaTime Core Location: C:\Users\David\AppData\Roaming\WakaTime\legacy-python-cli-master\wakatime\cli.py
CONFIG [WakaTime]: WakaTime Core Exists: true
CONFIG [WakaTime]: WakaTime Core Location: C:\Users\David\AppData\Roaming\WakaTime\legacy-python-cli-master\wakatime\cli.py
CONFIG [WakaTime]: WakaTime Core Exists: true
CONFIG [WakaTime]: Found python binary: C:\Users\David\AppData\Roaming\WakaTime\python\pythonw
CONFIG [WakaTime]: wakatime cli version check output: "13.1.0"
CONFIG [WakaTime]: wakatime cli version check exit code: 0
CONFIG [WakaTime]: Current cli version from GitHub: 13.1.0
INFO [WakaTime]: wakatime-cli is up to date.
CONFIG [WakaTime]: Python location: C:\Users\David\AppData\Roaming\WakaTime\python\pythonw
CONFIG [WakaTime]: CLI location: C:\Users\David\AppData\Roaming\WakaTime\legacy-python-cli-master\wakatime\cli.py
CONFIG [WakaTime]: API Key: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX0a8c
INFO [WakaTime]: Finished initializing WakaTime plugin.
INFO [WakaTime]: Checking for updates to WakaTime plugin...
```

**Comment by alanhamlett** [comment]
We'll fix this by switching to the new Go wakatime-cli, same as the Jetbrains plugin

https://github.com/wakatime/jetbrains-wakatime/blob/9e9b6982c0fe095e7a1b910b57571ae35ca3130c/src/com/wakatime/intellij/plugin/Dependencies.java#L233

**Comment by alanhamlett** [comment]
Fixed in v5.0.0.
